### PR TITLE
Adding Runbooks Publishing section

### PR DESCRIPTION
**Pull request by MarkSiedle**
This PR adds a publishing section to Runbooks and cleans up some limitations that have since been actioned.

### CODE REVIEW

File: docs/deployment-process/operations-runbooks/index.md [hunk 1]
```diff
@@ -57,14 +57,20 @@ You can **Run** a **Snapshot** as many times as you want to.
...
 
 ## Permissions
 
 Two permissions are available for the top-level Runbooks document: `RunbookView` (for viewing) and `RunbookEdit` (for creating, modifying, and deleting), so if you want to lock down the ability for your teams to create runbooks, you just need to disable these permissions.
 
 Once you create a runbook snapshot and run it, the typical `Deployment` and `Release` permissions are required.
 
+## Publishing
+
+Publishing allows a runbook author (users with `RunbookEdit` permissions) to nominate a Runbook Snapshot as being 'ready to run', informing others that the runbook process (at the given snapshot) has been tried and tested.
```
Comment (wordlee) [inline docs/deployment-process/operations-runbooks/index.md]: Is it worth say "tried and tested and is an approved version of the runbook." just to really drive it home :)

File: docs/deployment-process/operations-runbooks/index.md [hunk 2]
```diff
@@ -57,14 +57,20 @@ You can **Run** a **Snapshot** as many times as you want to.
...
+
+For consumers (users running a runbook), the option to run a published snapshot will be available on the Run screen. Users who do not have `RunbookEdit` permissions will have the published version selected by default, giving them some level of confidence that they will be running the expected snapshot.
```
Comment (wordlee) [inline docs/deployment-process/operations-runbooks/index.md]:
```suggestion
For consumers (users running a runbook), the option to run a published snapshot will be available on the Run screen. Users who do not have `RunbookEdit` permissions can access the published version by default, giving them some level of confidence that they will be running the expected snapshot.
```

File: docs/deployment-process/operations-runbooks/index.md [hunk 3]
```diff
@@ -91,4 +112,3 @@ For this early access feature, there are a number of things we decided to not in
 - Variable scoping to runbooks (there's no way to scope variables between your project's deployment process vs runbook processes).
 - Triggers.
```
Comment (wordlee) [inline docs/deployment-process/operations-runbooks/index.md]: Will triggers need to be removed from this list?

Comment (wordlee) [reply docs/deployment-process/operations-runbooks/index.md]: It is removed by #586 so never mind :)

File: docs/deployment-process/operations-runbooks/index.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ description: Runbooks can be used to automate routine or emergency operations-ce
 position: 140
 ---
 
-We're planning to introduce Runbooks in **Octopus 2019.10**.
+Runbooks were introduced in **Octopus 2019.10**.
```
Comment (NickJosevski) [inline docs/deployment-process/operations-runbooks/index.md]: 2019.11?

File: docs/deployment-process/operations-runbooks/index.md [hunk 5]
```diff
@@ -57,13 +57,34 @@ You can **Run** a **Snapshot** as many times as you want to.
...
+We don't believe that channels or lifecycles (progression) make sense for runbooks. Runbooks can be run on any environments you have access to. The interface has been designed to let you run a runbook quickly, so there's a single `Run...` screen where you choose environments and specify any `Advanced` options, and then you run it.
 
-## Permissions
+## Permissions and Roles
 
-Two permissions are available for the top-level Runbooks document: `RunbookView` (for viewing) and `RunbookEdit` (for creating, modifying, and deleting), so if you want to lock down the ability for your teams to create runbooks, you just need to disable these permissions.
+Permissions are available to help you manage access to Runbooks, these include `RunbookView` (for viewing), `RunbookEdit` (for creating, editing, deleting and snapshotting), and various `RunbookRun*` permissions.
```
Comment (NickJosevski) [inline docs/deployment-process/operations-runbooks/index.md]: > and various `RunbookRun*` 
maybe just list the 3?

could mention it follows the pattern fo the Deployment permissions

File: docs/deployment-process/operations-runbooks/index.md (hunk 4 above)
Comment (MarkSiedle) [reply docs/deployment-process/operations-runbooks/index.md]: Runbooks were introduced in 2019.10 behind this feature flag. 2019.11 is just where we are lifting the feature flag and tightening a few things.

Comment (NickJosevski) [reply docs/deployment-process/operations-runbooks/index.md]: But if you want docs to help drive a version a customer will pick to use a feature, should it not be when the feature ships completely, not in EAP?

Comment (MarkSiedle) [reply docs/deployment-process/operations-runbooks/index.md]: I'll update this sentence to reflect when we lifted the feature flag (2019.11) so it's clear

Comment (NickJosevski) [reply docs/deployment-process/operations-runbooks/index.md]: 👍

**Pull request merged**